utils/chain-ops/genesis/config.go

```diff
@@ -90,6 +90,8 @@ type DeployConfig struct {
 	FinalSystemOwner common.Address `json:"finalSystemOwner"`
 	// GUARDIAN account in the KromaPortal
 	PortalGuardian common.Address `json:"portalGuardian"`
+	// Owner of the Guardian Token
+	SecurityCouncilTokenOwner common.Address `json:"securityCouncilTokenOwner"`
```

Comment:
Don't we need to add this variable to `InitDeveloperDeployedAddresses()` below?

Reply:
Yes, this value is included in the deploy-config.